ocaml/util/ocamltest.ml

```diff
@@ -70,84 +61,8 @@ let assert_raises_match exception_match fn =
 let assert_raises expected =
   assert_raises_match (function exn -> exn = expected)
 
-let assert_raises_any f =
```

Comment:
Unless I'm misreading this function, it was broken anyway: it would always have succeeded.